frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx

```diff
@@ -96,8 +106,15 @@ export const TicketsList = ({ tickets }: TicketsListProps) => {
 		return () => ViewerService.off(VIEWER_EVENTS.BACKGROUND_SELECTED);
 	}, []);
 
+	const filterItems = (items, queries: string[]) => items.filter((ticket) => {
+		const templateCode = availableTemplates.find((template) => template._id === ticket.type).code;
+		const ticketCode = `${templateCode}:${ticket.number}`;
```

Comment:
Then you wouldn't be able to search for an entire ticket code. Say for instance you had a large list of tickets but knew the ticket code you wanted was 'RIS:14', you could type in 'RIS:14' and it will filter correctly, whereas with the other method your best option would be to do two separate searches for 'RIS' and '14' which I don't think is intuitive